### ISSUE TITLE
Index Module Installer

### ISSUE THREAD

**Issue by Facundo-Barbera** [Facundo-Barbera/Python-Creations#4]
- Would it be good if I made the Index download any python modules that are needed thru pip?

**Comment by Facundo-Barbera** [comment]
@jclopeztavera What do you think?

**Comment by jclopeztavera** [comment]
Good idea, you should take a look a the pip docs; in particular, the requirements.txt format:
https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format

**Comment by Facundo-Barbera** [comment]
I don't fully understand this, I've read it thru, but I don't get if it's the installation requirements for pip or just a readme format

**Comment by Facundo-Barbera** [comment]
I haven't been able to use pip install internally from a py file, which I believe will be a big issue when using modules
